src/index.ts

```diff
@@ -96,12 +114,20 @@ class MultinetAPI {
     return this.client.get('workspaces');
   }
 
-  public workspace(workspace: string): Promise<string> {
+  public getWorkspacePermissions(workspace: string): Promise<WorkspacePermissionsSpec> {
+    if (!workspace) {
+      throw new Error('argument "workspace" must not be empty');
+    }
+
+    return this.client.get(`workspaces/${workspace}/permissions`);
+  }
+
+  public setWorkspacePermissions(workspace: string, permissions: WorkspacePermissionsSpec): Promise<WorkspacePermissionsSpec> {
     if (!workspace) {
       throw new Error('argument "workspace" must not be empty');
     }
 
-    return this.client.get(`workspaces/${workspace}`);
+    return this.client.axios.put(`workspaces/${workspace}/permissions`, permissions);
```

Comment:
Can you add a `.put()` method on the Client class so that this call is of the same kind as the `.get()` and `.post()` calls throughout this file?

There's a discussion we should have as to whether the Client class is even necessary, but until then I'd like to keep things consistent to avoid future pain.

Reply:
> Can you add a `.put()` method on the Client class so that this call is of the same kind as the `.get()` and `.post()` calls throughout this file?

Sure

> There's a discussion we should have as to whether the Client class is even necessary, but until then I'd like to keep things consistent to avoid future pain.

Sounds good.

Reply:
1f5d1c1